summarization/hf_transformers/notebooks/project-metadata.ipynb

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I think we can remove this notebook. Doesn't really do much wrt the integration

Reply:
Done!